app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java

```diff
@@ -251,6 +256,18 @@ public void onActivityCreated(Bundle savedInstanceState) {
         // Subtitles font size configuration
         binding.subtitlesText.setTextSize(subtitlesSize);
 
+        // Subtitles font position (margin bottom)
+        if (subtitlesPosition>0) {
+            ViewGroup.MarginLayoutParams currentLayoutParams = (ViewGroup.MarginLayoutParams)binding.subtitlesText.getLayoutParams();
+            currentLayoutParams.bottomMargin=(48+Utils.convertDpToPixel(requireContext(), subtitlesPosition));
```

Comment:
```suggestion
            currentLayoutParams.bottomMargin = ( 48 + Utils.convertDpToPixel(requireContext(), subtitlesPosition));
```

Also, where does the 48 come from?